server/fixity.go

```diff
@@ -139,3 +155,118 @@ func (s *RESTServer) scanfixity() {
 	}
 	log.Println("Ending scanfixity. duration = ", time.Now().Sub(starttime))
 }
+
+func (s *RESTServer) GetFixityHandler(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+
+	item := r.FormValue("item")
+	start := r.FormValue("start")
+	end := r.FormValue("end")
+	status := r.FormValue("status")
+
+	log.Println("GetFixityHandler called")
+	log.Println("item= ", item)
+	log.Println("start= ", start)
+	log.Println("end= ", end)
+	log.Println("status= ", status)
```

Comment:
These log lines should be deleted or commented out.

Reply:
Done.